docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,21 @@
+# Local Development
+
+Tekton docs are hosted on [Netlify](https://docs.netlify.com), making use of [Hugo](https://gohugo.io/documentation/) for content management.
+
+## Install developer tools
+
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [hugo](https://gohugo.io/getting-started/installing/)
+- [netlify dev](https://docs.netlify.com/cli/get-started/#installation)
+
+## Installing website dependencies
+
+```sh
+npm install
+```
+
+## Running local instance
+
+```sh
+netlify dev
+```
```

Comment:
why are these docs so different from https://github.com/tektoncd/website/blob/master/DEVELOPMENT.md and would it make sense to link to https://github.com/tektoncd/website/blob/master/DEVELOPMENT.md from here?